Annotato/Annotato/ViewModels/Annotation/AnnotationViewModel.swift

```diff
@@ -37,24 +38,45 @@ class AnnotationViewModel: ObservableObject {
     @Published private(set) var isMinimized = true
     @Published private(set) var modelWasUpdated = false
 
+    @Published private(set) var isResolving = false
+    private(set) var resolveBySave = false
+    private(set) var conflictIdx: Int?
+
     init(
         model: Annotation,
         document: DocumentViewModel,
         webSocketManager: WebSocketManager?,
-        palette: AnnotationPaletteViewModel? = nil
+        palette: AnnotationPaletteViewModel? = nil,
+        isResolving: Bool = false,
+        conflictIdx: Int? = nil
```

Comment:
Is there a need for two separate stored properties? In other words, will we ever have `conflictIdx != nil && !isResolving` or `conflictIdx == nil && isResolving`? If not we can just have `conflictIdx` as a stored property and if we really have to, make the other a computed property

Reply:
We can even try to make the paletteViewModel a computed property, but more importantly let's try to fix the 0 width 0 height palette view model first.

Reply:
Ok I did the first one already